grails-forge-core/src/main/resources/pom.xml

```diff
@@ -20,7 +20,7 @@
         <dependency>
             <groupId>org.grails</groupId>
             <artifactId>grails-gradle-plugin</artifactId>
-            <version>6.2.1</version>
+            <version>6.2.2-SNAPSHOT</version>
```

Comment:
Looking at the history, I don't think this should be updated to SNAPSHOT.

Reply:
Hmm, it seems the tests passed. Lets push it and see if we get `6.2.2-SNAPSHOT` as an option in the UI.